ci/container_index/lib/utils.py

```diff
@@ -0,0 +1,79 @@
+import yaml
+from datetime import datetime, date
+from glob import glob
+from os import environ, path, mkdir, unsetenv, listdir,\
+    unlink, devnull, getenv, getcwd, chdir, system
+from shutil import rmtree
+from subprocess import check_call, CalledProcessError, STDOUT
+
+
+def execute_command(cmd):
+    """Execute a specified command"""
+    try:
+        fnull = open(devnull, "w")
+        check_call(cmd, stdout=fnull, stderr=STDOUT)
+        return True
+    except CalledProcessError:
+        return False
+
+
+def load_yaml(file_path):
+    data = None
+    try:
+        with open(file_path, "r") as f:
+            data = yaml.load(f)
+    except Exception as e:
+        pass
```

Comment:
so if yaml is not valid a file, we do nothing to report it, refer to my earlier review comment where I pointed out about doing error check while loading user provided yaml files.

Reply:
Fixing this